sections/03_categorize.md

```diff
@@ -196,55 +196,73 @@ this field.
-in time and money [@doi:10.1016/j.ijmedinf.2016.09.014]. Due to resource
-limitations,  many existing works in this direction (including many referenced
-in this review)  skipped the expert review process, which may face great
-skepticism from  medical domain experts. To date, even the most resourceful
-large national consortia  have difficulty acquiring enough expert-validated
-labeled data. For instance,  in the eMERGE consortia and PheKB database
-[@url:https://phekb.org/implementations],  most sample sizes with expert validations
-are between 100 to 300 patients,  which is quite small even for simple machine
-learning, not to mentioned data-hungry  deep learning models.
-
-This issue also common to many related domains -- around medical images, omics.
-(thus the motivation for semi-supervised learning as briefly mentioned in
-earlier sections)
-
-In summary, this subsection emphasize on two important aspects that are not
-addressed by other sections:  1) Data (and sometimes learnt
-features/representations)  need to be validated by clinicial experts.   2) The
-amount of expert labels needs to be sufficient large.
+Lack of true labels is perhaps among the biggest obstacles for EHR-based
+analyses that employ machine learning (e.g., phenotyping). Popular deep
+learning (and machine learning) methods are often used to tackle classification
+tasks and thus require ground-truth labels for training.  For EHRs,
+unfortunately, this means that researchers need to hire multiple clinicians to
+manually read and annotate individual patients' records through a process called
+chart review. This allows researchers to assign "true" labels, i.e. those that
+match our best available knowledge. Depending on the application, sometimes the
+features constructed by algorithms also need to be manually validated and
+interpreted by clinicians. This can be time consuming and expensive
+[@doi:10.1016/j.ijmedinf.2016.09.014]. Because of these costs, much of this
+research, including the work cited in this review, skips the process of expert
+review. Clinicians' skepticism for research without expert review may greatly
+dampen their enthusiasm for the work and consequently reduce its impact. To
+date, even well-resourced large national consortia have been challenged by the
+task of acquiring enough expert-validated labeled data. For instance, in the
+eMERGE consortia and PheKB database [@url:https://phekb.org/implementations],
+most samples with expert validation contain only 100 to 300 patients. These
+datasets are quite small, even for simple machine learning algorithms. The
+challenge is greater for deep learning models with many parameters. While
+unsupervised and semi-supervised approaches can help with small sample sizes,
+the field would benefit greatly from large collections of anonymized records in
+which a substantial number of records have undergone expert review.
+
+This challenge is not unique to EHR-based studies. Work on medical images,
```

Comment:
I think we can leave this section as-in, and we'll likely want to refer back to in in our Discussion.  As you say here, data limitations are prevalent in most of the domains we're covering.